plugins/toolbox-search/test/index.ts

```diff
@@ -37,7 +37,7 @@ document.addEventListener('DOMContentLoaded', function () {
     toolbox,
   };
   createPlayground(
-    document.getElementById('root'),
+    document.getElementById('root')!,
```

Comment:
Can you change this to check if `'root'` exists instead of asserting it exists?

Reply:
done!